### ISSUE TITLE
Dutch Translation

### ISSUE THREAD

**Issue by Haringstad** [ntilley905/faastatus#14]
```
{
    "config": {
        "abort": {
            "already_configured": "Dit vliegveld is al geconfigureerd."
        },
        "error": {
            "invalid_airport": "Vliegveld-code is niet valide"
        },
        "step": {
            "user": {
                "data": {
                    "id": "Vliegveld"
                }
            }
        }
    },
    "title": "FAA Delays"
}

```

**Comment by ntilley905** [comment]
Thanks for this! Sorry for my delayed response, I've been a bit busy. I appreciate the contribution, feel free to open a pull request against this repo or I can drop it in as well. This repository will soon be archived though as (hopefully) this component will be merged into the actual Home Assistant core repo as well. In that component the translations changed a bit too as I added a bit more detail to the configuration step. Feel free to contribute this over there as well!

If for some reason the HA component doesn't get merged and if you would prefer me to bring in the translation manually I'll certainly do that, although I'll give you a couple more phrases as I'll update the full strings here soon.

**Comment by Haringstad** [comment]
Hi, please let me know where I can find the component in the "ha tree", and
I'll make some more contributions for translations?
And yes, I am busy myself too, so if you could "drop it in" for me, I
would appreciate it!

Regards,

Jacco van Koll

**Comment by ntilley905** [comment]
The HA pull request is [here](https://github.com/home-assistant/core/pull/41347), it has been developer reviewed, just waiting on a merge. I can add in your translations as a new commit pre merge or wait until it has been merged and either one of us can open a pull request against it to add them. 

The new strings file is [here](https://github.com/home-assistant/core/blob/07386fd44a338efa7e433ddbcb7dc45bc9588cfc/homeassistant/components/faadelays/strings.json), the string that has been added is "Enter a US Airport Code in IATA Format".

**Comment by Haringstad** [comment]
DUTCH:

{
  "title": "FAA Delays",
  "config": {
    "step": {
      "user": {
        "title": "FAA Delays",
        "description": "Voer een US vliegveld Code in IATA Format in",
        "data": {
          "id": "Vliegveld"
        }
      }
    },
    "error": {
      "invalid_airport": "Vliegveld code is niet valide",
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
    },
    "abort": {
      "already_configured": "Dit vliegveld is al geconfigureerd."
    }
  }
}

GERNMAN:

{
  "title": "FAA Delays",
  "config": {
    "step": {
      "user": {
        "title": "FAA Delays",
        "description": "Geben Sie einen US-Flughafencode im IATA-Format
ein",
        "data": {
          "id": "Flughafen"
        }
      }
    },
    "error": {
      "invalid_airport": "Flughafen code ist ungültig"
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
    },
    "abort": {
      "already_configured": "Dieser Flughafen ist bereits konfiguriert."
    }
  }
}


I am up to my eyeballs in work atm...... Sorry! But here are 2 translations
already. Dutch and German

Regards,

Jacco van Koll

**Comment by ntilley905** [comment]
No apologies needed, thank you so much for the help! As soon as this gets merged over in Home Assistant I'll add these translations.

**Comment by Haringstad** [comment]
As far as I can see, that might happen very soon! Second review, so.....

Regards,

Jacco van Koll
